### DISCUSSION TITLE
Installed 'one-click-deploy' project locally, but can't deploy it to Northflank

### DISCUSSION THREAD

**Discussion by martinbrud** [vendurehq/vendure#2673] [General]
Hi,

I have installed https://github.com/vendure-ecommerce/one-click-deploy project locally and I have it up and running fine. I have tried in a lot of ways to "connect" it to a project on Northflank in order to get a CI/CD setup so when I commit a change to Github then it triggers build and deploy on the Northflank side.

I can also install a project on Northflank from one of the templates on https://docs.vendure.io/guides/deployment/deploy-to-northflank/ and it runs fine on Northflank . But can't get that up running locally and connect to CI/CD.

Can anyone point me in the right direction?

**Comment by michaelbromley** [comment]
Hi,

Since you already got the one-click-deploy repo running locally, you should then be able to:

- Create a new repo on GitHub and push your local one-click-deploy project to that repo
- Then use the template from the guide, and change the `"projectUrl": "https://github.com/vendure-ecommerce/one-click-deploy",` to point at your newly-created repo url.
